src/Microsoft.AspNetCore.Razor.Language/CodeGeneration/DefaultDocumentWriter.cs

```diff
@@ -143,7 +143,12 @@ public override void VisitNamespaceDeclaration(NamespaceDeclarationIntermediateN
 
             public override void VisitClassDeclaration(ClassDeclarationIntermediateNode node)
             {
-                using (Context.CodeWriter.BuildClassDeclaration(node.Modifiers, node.ClassName, node.BaseType, node.Interfaces))
+                using (Context.CodeWriter.BuildClassDeclaration(
+                    node.Modifiers,
+                    node.ClassName,
+                    node.BaseType,
+                    node.Interfaces,
+                    node.TypeParameters.Select(p => p.ParameterName).ToArray()))
```

Comment:
Suggest putting this in a variable. Easier for debugging.

Reply:
👎 I don't like this trend. Optimize for readability.